docs/Configuration.md

```diff
@@ -115,7 +115,7 @@ An array of source extensions to include in the bundle. For example, if you woul
 
 #### `resolverMainFields`
 
-Type: `Array<string>`
+Type: `Array<string>` (default: ['browser', 'main'])
```

Comment:
I don't think we use this style of documenting defaults elsewhere in the docs. Would you mind mentioning this default in the descriptive text below?